packages/builder/babel/presets.js

```diff
@@ -1,14 +1,17 @@
 /* eslint-disable no-console */
+// const path = require('path');
+const path = require('path');
 
 const presets = [
   require.resolve('@babel/preset-env'),
   require.resolve('@babel/preset-react'),
 ];
 
-// Try to load `@theforeman/env/babel`
+// Try to load `@theforeman/env/babel` from the consumer
 try {
-  const { paths } = require.main;
-  const envPreset = require.resolve('@theforeman/env/babel', { paths });
+  const envPreset = require.resolve('@theforeman/env/babel', {
```

Comment:
The reason why `@theforeman/env` is not in dependencies of `@theforeman/builder` is that we are counting on its presence because it is a `foreman` dependency, right?

Reply:
Right, and using the `env` depends on the consumer choice based on the scenario.

Reply:
I made some tweaks so it would be more readable